operator/src/test/resources/runDeployedTests.sh

```diff
@@ -0,0 +1,15 @@
+cd ../../..
+eval $(minikube docker-env)
+kubectl create namespace keycloak-test
```

Comment:
I understood that the tests are already creating a specific namespace to test things in.

Reply:
afaik quarkus will deploy the resources but in an already created namespace... and this is for the remote test, that needs to have the namespace created before the test starts ( as it needs the operator to be deployed )